package-lock.json

```diff
@@ -1,6009 +1,8 @@
 {
   "name": "@firebolt-js/manage-sdk",
   "version": "0.1.0-alpha.4",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
This gets annoying, but make sure you have an up to date npm so it doesn't use older lockfile versions